app/src/main/java/com/uncookthebook/app/MainActivity.java

```diff
@@ -145,6 +156,26 @@ public GoogleSignInAccount getGoogleAccount() {
     @Override
     public void setGoogleAccount(GoogleSignInAccount account) {
         userAccount = account;
+
+        // Use case of APIServiceClient
+        APIService apiServiceClient = APIServiceUtils.getAPIServiceClient();
+        User user = new User(account.getId(), account.getGivenName(),
+                account.getFamilyName(), account.getEmail());
+        apiServiceClient.addUser(new TokenizedObject<>(account.getIdToken() + "casda", user)).enqueue(new Callback<String>() {
```

Comment:
what's the purpose of casda

Reply:
to make the token check fail so that I could see the failing message

Reply:
That's a use case. It is not the final code so it has to be removed